src/plugins/ramps/infinite/infiniteRampPlugin.ts

```diff
@@ -189,6 +194,15 @@ export const infiniteRampPlugin: RampPluginFactory = (
           return { supported: false }
         }
 
+        // Global constraints pre-check
+        const paymentTypes: FiatPaymentType[] = ['wire']
```

Comment:
Make `'wire'` a constant at the top of the module and use it here and in the RampQuote instances